data/tooltips.txt

```diff
@@ -641,6 +737,18 @@ tip "firing slowing / shot:"
 tip "firing disruption / shot:"
 	`Each shot disrupts the target's shields, allowing hostile weapon damage to "leak through" to the hull even if shields are up. Shield disruption wears off over time.`
 
+tip "firing discharge / shot:"
+	`Amount of shield damage over time by this weapon per shot when firing.`
+
+tip "firing corrosion / shot:"
+	`Amount of hull damage over time by this weapon per shot when firing.`
+
+tip "firing leak / shot:"
+	`Amount of fuel damage over time by this weapon per shot when firing.`
+
+tip "firing burn / shot:"
+	`Amount of heat damage over time by this weapon per shot when firing.`
```

Comment:
These should maybe parrot the not-over-time tooltips more closely, e.g. "Heat damage incurred per shot."

Reply:
Do these look good now?